openpower-build-setup.sh

```diff
@@ -25,10 +25,21 @@ if [[ "${distro}" == fedora ]];then
   Dockerfile=$(cat << EOF
 FROM fedora:latest
 
-RUN dnf --refresh upgrade -y && \
-	dnf install -y vim gcc-c++ flex bison git ctags cscope expat-devel \
-	patch zlib-devel zlib-static perl
-RUN groupadd -g ${GROUPS} ${USER} && useradd -d ${HOME} -m -u ${UID} -g ${GROUPS} ${USER}
+RUN dnf --refresh upgrade -y && dnf install -y \
+	vim \
+	gcc-c++ \
+	flex \
+	bison \
+	git \
+	ctags \
+  	cscope \
+	expat-devel \
+	patch \
+	zlib-devel \
+	zlib-static perl
```

Comment:
ahh yeah, we don't actually build with fedora yet I don't think, so we probably should update that if we're going to, which we should. I'll push an update, thanks.